### PR TITLE
Install scripts without adding gem

### PR DESCRIPTION
**Pull request by sweatC**
Need to add ability to use the template without add gem instead I would like to have:

`rails app:template LOCATION=https://raw.githubusercontent.com/jetthoughts/jt_tools/master/template.rb`

There are some examples: 

* https://github.com/excid3/jumpstart/blob/master/template.rb#L4
* https://github.com/mattbrictson/rails-template

More docs: https://guides.rubyonrails.org/rails_application_templates.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,12 @@
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Open your project and then run:
 
-```ruby
-gem 'jt_tools', groups: [:development]
+```bash
+rails app:template LOCATION=https://raw.githubusercontent.com/jetthoughts/jt_tools/master/template.rb
```
Comment (pftg) [inline README.md]: we need to support both ways as gem and as remote

File: template.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+copy_file '.pronto_eslint_npm.yml'
+copy_file '.rubocop.yml'
+copy_file '.yamllint'
+
+say 'Copying circleci configuration'
+directory 'circleci', '.circleci'
+
+say 'Copying heroku configuration'
+copy_file 'app.json'
+copy_file 'Procfile'
```
Comment (pftg) [inline template.rb]: we does not require to copy everything to root! please use the same strucutre as it was. we still should support gem strucutre too

**Comment by sweatC** [comment]
@pftg pls double-check.

File: .better-html.yml [hunk 3]
```diff
@@ -1 +0,0 @@
-lib/install/.better-html.yml
```
Comment (pftg) [inline .better-html.yml]: this is symlink files no need to change them

**Comment by sweatC** [comment]
@pftg fixed sym links. Tested and all works nice.

**Comment by pftg** [comment]
Please remove duplication: we have 2 template.rb I root and in lib/install/template.rb.

Could we have one file?

**Comment by sweatC** [comment]
@pftg, for now, I didn't find a way to make this in one file.

**Comment by pftg** [comment]
@sweatC just update rake task to use root template instead of install version

**Comment by pftg** [comment]
What it will cause?

**Comment by SviridovSV** [comment]
@pftg Please, check https://github.com/jetthoughts/jt_tools/pull/1/commits/724f71d876f0aa020eafa0706977dfc14949580e
Is it what you meant?

**Comment by pftg** [comment]
@SviridovSV there is references to old file: https://github.com/jetthoughts/jt_tools/blob/master/lib/tasks/install.rake#L7

But I would like to use old file and extend it instead of adding template to root. But if it's not possible then use template from root, but need to delete old file and replace refrences to new one.

**Comment by pftg** [comment]
@SviridovSV could you extend lib template instaed of add new template to root

**Comment by SviridovSV** [comment]
@pftg Please check.

**Pull request merged**